config/dyadic_ekm_security_provider.yml

```diff
@@ -0,0 +1,20 @@
+# Cloud Foundry Java Buildpack
+# Copyright 2013-2016 the original author or authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Configuration for the Luna Security Provider framework
+---
+version: 1.+
+repository_root: https://repo.dyadicsec.com/cust/pcf
+logging_enabled: false
```

Comment:
I don't believe that you've added support for configuring logging, so this entry can be removed.